### ISSUE TITLE
Threads

### ISSUE THREAD

**Issue by randygrolemund** [fireice-uk/xmr-stak#1706]
Have you guys thought about adding dual threads to your auto generated amd.txt file? I'm sure you know how much it improves performance on most algo's.

Would you be interested in creating an exclusive version for me, with some of the additions I would like to see in the next release? Maybe we can work out a price? Either that, or I need to teach myself another language. haha

**Comment by psychocrypt** [comment]
Sry for the late response. What do you mean with dual thread? It is possible to add two entries in the amd.txt file per GPU to use two threads on one GPU.
Is this what you request?

**Comment by pcguru** [comment]
I think he want to be able to specify "threads": 2 per gpus instead of copying the config for each gpu. Sgminer has this feature for example. Not a killer feature really.

**Comment by randygrolemund** [comment]
Correct..... sorry for late response guys, been busy. Or just have it automatically do that. Stak does a pretty good job of auto detecting the best settings.... if it could do that while also using dual threads.... it would be awesome. As you know.... I wrote a profit manager based on Stak. Everyone who uses my software, uses Stak. I would like to be able to make their life easier (thus my software), but automating  most of the config.

**Comment by randygrolemund** [comment]
My profit manager is cryptonight only, and all alt-coins. 

https://github.com/randygrolemund/shitcoin-profitmanager/releases

![image](https://user-images.githubusercontent.com/8581255/43317854-2d4f4bb2-9164-11e8-95bf-8a6f43992284.png)
